src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -0,0 +1,313 @@
+    },
+    toolbarGutters: {
+        padding: `0 ${theme.spacing(2)}px`,
+    },
+    placementOfList: {
+        position: 'absolute',
+        bottom: theme.spacing(3),
+        right: theme.spacing(3),
+        [theme.breakpoints.down('xs')]: {
+            bottom: 0,
+            right: 0,
+            width: '100%',
+        },
+    },
+    infoList: {
+        marginBottom: theme.spacing(2),
+        '&:last-child': {
+            marginBottom: 0,
+        },
+    },
+    subTitle: {
+        color: Colors.black[200],
+    },
+    snackbar: {
+        position: 'inherit',
+        transform: 'none',
+    },
+    SnackbarContent: {
+        borderRadius: 0,
+        padding: `0 ${theme.spacing(2)}px 0 ${theme.spacing(1)}px`,
+    },
+    messageContainer: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+    messageTextContainer: {
+        marginRight: `${theme.spacing(3)}px`,
+    },
+}));
+const createFileItem = (increment: number): FolderItem => ({
+    id: increment,
+    open: true,
+    duration: null,
+    buttonLabel: 'Close',
+    name: 'PX Blue is Awesome.pdf',
+    progress: 0,
+    status: `Uploading (0%)`,
+});
+
+export const ProgressBar = (): JSX.Element => {
+    const dispatch = useDispatch();
+    const classes = useStyles();
+    const [fileUploadList, setFileUploadList] = useState<FolderItem[]>(uploadFileList);
+
+    const [radioButtonvalue, setRadioButtonvalue] = useState('1');
+    const [count, setCount] = useState(0);
+    const changeRadioGroup = (event: React.ChangeEvent<HTMLInputElement>): void => {
+        setRadioButtonvalue((event.target as HTMLInputElement).value);
+    };
+
+    const uploadFile = useCallback((): void => {
+        setCount(count + 1);
+        setFileUploadList((oldList) => [...oldList, createFileItem(count)]);
+    }, [fileUploadList, setFileUploadList]);
+
+    const removInfoList = useCallback(
+        (index: number, status: string): void => {
+            if (status === 'Complete') {
+                return;
+            }
+            setFileUploadList((oldList) => oldList.filter((item) => item.id !== index));
+        },
+        [fileUploadList]
+    );
+
+    const handleRequestClose = useCallback((event: any, reason: string, id: number) => {
+        if (reason === 'clickaway') {
+            return;
+        }
+        setFileUploadList((oldList) => oldList.filter((item) => item.id !== id));
+    }, []);
+
+    useEffect(() => {
+        const progressInterval = setInterval(() => {
+            const newList = [...fileUploadList];
+            for (let i = 0; i < fileUploadList.length; i++) {
+                if (fileUploadList[i].progress < 100) {
+                    const newItem: FolderItem = {
+                        ...fileUploadList[i],
+                        progress: fileUploadList[i].progress + 1,
+                        status: `Uploading (${fileUploadList[i].progress + 1}%)`,
+                    };
+                    newList[i] = newItem;
```

Comment:
Add a little randomness so they don't all upload in the same amount of time.
```suggestion
                    const newPercent = Math.min(100, fileUploadList[i].progress + Math.ceil(Math.random()*5))
                    const newItem: FolderItem = {
                        ...fileUploadList[i],
                        progress: newPercent,
                        status: `Uploading (${newPercent}%)`,
                    };
                    newList[i] = newItem;
```